src/state/reducers/userReducer.js

```diff
@@ -1,14 +1,26 @@
 // export const USER_EVENT_START = 'USER_EVENT_START';
-import { USER_PROFILE } from '../actions/index';
+import * as a from '../actions/index';
 
 const initialState = {
+  authState: {},
+  authService: {},
+  userInfo: null,
+  isFetching: false,
   photo: '',
   profile: '',
```

Comment:
At this point, can we delete `photo` and `profile` from state? It seems all profile data should be handled in `userInfo` and we aren't currently storing photos of users.